Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -13055,7 +13056,19 @@ def get_query_entries_by_id_request(job_id: str) -> Dict[str, Any]:
         Dict[str,Any]: a dictionary of the raw entries linked to the Job ID
     """
     params = assign_params(key=API_KEY, type='log', action='get', job_id=job_id)
-    return http_request(URL, 'GET', params=params)
+    
+    # if the job has not finished, wait for 1 second and try again (until success or max retries)
+    for try_num in range(1, GET_LOG_JOB_ID_MAX_RETRIES + 1):
+        response = http_request(URL, 'GET', params=params)
+        status = response.get('response', {}).get('result', {}).get('job', {}).get('status', '')
+        demisto.debug(f'Job ID {job_id}, response status: {status}')
+        demisto.debug(f'{response=}')
+        if status == 'FIN':
+            return response
+        else:
+            demisto.debug(f'{try_num=}. Job not completed, Retrying in 1 second...')
+            time.sleep(1)
```

Comment:
@yuvalbenshalom - would appreciate your opinion here about using the 1 sec sleep. 
The logic here is kind of internal polling which is limited by the GET_LOG_JOB_ID_MAX_RETRIES.
Please advice.

Reply:
@samuelFain this can be problematic. if we may wait 10+ sec, considering other delays in the fetch, we may hit the 60 sec in some cases.

why don't we save the Ids in the context and re try them in the next fetch? we are doing a similar flow in splunkpy when getting drill down on fetch. you can check this out.

Reply:
Due to the short job life, the context solution wont work for longer fetch intervals.
For now we will handle this by lowering the max retries value.